lms/templates/problem_notifications.html

```diff
@@ -0,0 +1,22 @@
+<%page expression_filter="h" args="notification_type, notification_icon, notification_message, hint_notification"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+<%
+    notification_type = "info"
+    notification_icon = "question"
+    notification_message = "Hint (1 of 2): A fruit is the fertilized ovary from a flower. Hint (2 of 2): A fruit contains seeds of the plant."
+    hint_notification = True
+%>
+
+<div class="notification ${notification_type}">
+    <span class="icon fa fa-${notification_icon}" aria-hidden="true"></span>
```

Comment:
@cptvitamin we do have copy in the notification that is explicit. If we add "Error" SR text, would it end up like Error - Incorrect: (0/4 points earned)?
 
See example:
https://alisan617.sandbox.edx.org/courses/course-v1:edX+DemoX+Demo_Course/courseware/1414ffd5143b4b508f739b563ab468b7/workflow/